nodeup/pkg/model/kube_apiserver_test.go

```diff
@@ -70,53 +71,63 @@ type fakeKeyStore struct {
 	T *testing.T
 }
 
+var _ fi.Keystore = &fakeKeyStore{}
```

Comment:
Not actually necessary, as this file is assigning it to a field of type `CAStore` and a `CAStore` embeds `Keystore`.

But perhaps this type should be pulled out into a separate file, since it is referenced from tests other than kube_apiserver?

Reply:
I like to leave the assignment because it documents that the struct should implement the interface (i.e. it is for reading).  That could reveal my language history though ;-)

Agree that it can be pulled out into a separate file - did that, and added some comments, including on the split.